README.md

```diff
@@ -3,6 +3,12 @@ Restricted Input
 
 Allow restricted character sets in `input` elements.
 
+## Demo
+
+Try the latest version of Restricted Input:
+
+http://braintree.github.io/restricted-input/
```

Comment:
Could you change this to:

```
Try the latest version of Restricted Input [here](https://braintree.github.io/restricted-input/).
```

Note that it's an "inline" link and that it's `https`.